ETC.tsv

```diff
@@ -11551,7 +11551,7 @@ ETC_20150428_011551	The helper's level is higher than the character level.
 ETC_20150428_011552	The helper has been hired.
 ETC_20150428_011553	The helper was let go.
 ETC_20150428_011554	There is a helper already hired.
-ETC_20150428_011555	It can be only registered through the Magic Association NPC.
+ETC_20150428_011555	It can be only registered through the Mage Society NPC.
```

Comment:
It's a pretty generic name for a group, so there's a hundred ways to write it. 

'Mage's Association' won't work, since the name clashes with a certain other franchise. Other versions are also taken.